projects/max14906/src/common/common_data.h

```diff
@@ -49,7 +49,10 @@
 
 extern struct no_os_uart_init_param max14906_uart_ip;
 extern struct no_os_spi_init_param max14906_spi_ip;
-extern struct no_os_gpio_init_param max14906_fault_gpio_param;
 extern struct max149x6_init_param max14906_ip;
 
+#ifdef BASIC_EXAMPLE
+extern struct no_os_gpio_init_param max14906_fault_gpio_param;
```

Comment:
is this change related to adding stm32 support?

Reply:
Not really, will revert the change.